src/ProjectFileManager.cpp

```diff
@@ -1017,11 +1017,8 @@ AudacityProject *ProjectFileManager::OpenProjectFile(
       selectionManager.SSBL_SetFrequencySelectionFormatName(
       formats.GetFrequencySelectionFormatName());
       selectionManager.SSBL_SetBandwidthSelectionFormatName(
-      formats.GetBandwidthSelectionFormatName());
-
-      SelectionBar::Get( project )
-         .SetRate( ProjectRate::Get(project).GetRate() );
-
+      settings.GetBandwidthSelectionFormatName());
```

Comment:
The second commit in the sequence breaks the build with this line, at it is fixed at "SelectionBarListener no longer controls snapping"

It would be nicer to have a commit sequence in which all build

Reply:
Bad merge with master, sorry for that